### PR TITLE
Add tests for jobs addition

### PR DESCRIPTION
**Pull request by shayakrach**

### CODE REVIEW

**Review by assafg95 — APPROVED** [review]

**Review by AshkenYariv — APPROVED** [review]
Approved!

File: conftest.py [hunk 1]
```diff
@@ -99,3 +120,24 @@ def job_2(user_HR):
...
+def valid_job_data(user_HR):
+    job_data = {'publisher': user_HR,
+                'company_name': DEFAULT_COMPANY_NAME,
+                'title': 'Software Engineer',
+                'location': DEFAULT_JOB_LOCATION,
+                'description': 'Part time software engineer at Wix: Back end',
+                'post_until': '2022-12-23',
+                'is_active': True,
+                'apply_link': DEFAULT_APPLY_LINK}
+    return job_data
```
Comment (ormergi) [inline conftest.py:226]: Why wont you use `Job` class instead?

**Review by ormergi — CHANGES_REQUESTED** [review]
Hi, please see my review on https://github.com/redhat-beyond/Side-Hustle/pull/78 first

File: conftest.py (hunk 1 above)
Comment (ormergi) [reply conftest.py:226]: Following our discussion offline about #78 (https://github.com/redhat-beyond/Side-Hustle/pull/78#discussion_r776848574), please try to add a class method to Job that converts it to dict.

**Review by ormergi — CHANGES_REQUESTED** [review]
Move the fixtures you added on conftest.py to where you use them.

File: jobs/tests/test_add_job_view.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    job = form.save()
+    assert Job.objects.filter(pk=job.id).exists()
+
+
+@pytest.mark.django_db
+def test_sign_up_invalid_data(invalid_posts_until_job_data):
+    for invalid_post_until_job_data in invalid_posts_until_job_data:
+        form = JobForm(data=invalid_post_until_job_data)
+        assert not form.is_valid()
```
Comment (ormergi) [inline jobs/tests/test_add_job_view.py]: I think the test is not relevant

File: jobs/tests/test_add_job_view.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    form = response.context["form"]
+    assert isinstance(form, JobForm)
+
+    form_initial_data = response.context["form"].initial
+    assert all(form_initial_data[key] == valid_job_data[key] for key in form_initial_data)
+
+
+@pytest.mark.django_db
+def test_add_valid_job(valid_job_data):
```
Comment (ormergi) [inline jobs/tests/test_add_job_view.py]: Please add another test with invalid job

**Review by ormergi — CHANGES_REQUESTED** [review]
Please move all the fixtures you added to conftest.py to where you use them.

**Review by ormergi — APPROVED** [review]

**Pull request merged**